pkg/ctl/create/iamserviceaccount_test.go

```diff
@@ -10,62 +12,65 @@ import (
 )
 
 var _ = Describe("create iamserviceaccount", func() {
-	DescribeTable("create service account successfully",
-		func(args ...string) {
-			commandArgs := append([]string{"iamserviceaccount"}, args...)
-			cmd := newMockEmptyCmd(commandArgs...)
-			count := 0
-			cmdutils.AddResourceCmd(cmdutils.NewGrouping(), cmd.parentCmd, func(cmd *cmdutils.Cmd) {
-				createIAMServiceAccountCmdWithRunFunc(cmd, func(cmd *cmdutils.Cmd, overrideExistingServiceAccounts bool) error {
-					Expect(cmd.ClusterConfig.Metadata.Name).To(Equal("clusterName"))
-					Expect(cmd.ClusterConfig.IAM.ServiceAccounts[0].Name).To(Equal("serviceAccountName"))
-					Expect(cmd.ClusterConfig.IAM.ServiceAccounts[0].AttachPolicyARNs).To(ContainElement("dummyPolicyArn"))
-					count++
-					return nil
+	saNames := [2]string{"sa-1", "default"}
+	for i := range saNames {
```

Comment:
Why the refactor of this DescribeTable?

I think you meant to add an `Entry` to this?